pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>common</artifactId>
-        <version>6.1.12</version>
+        <version>6.1.16-14</version>
```

Comment:
I don't think we should be using nano version artifacts for connector releases. They are typically meant for cloud-based projects.